lib/marloss/store.rb

```diff
@@ -2,14 +2,16 @@
 
 module Marloss
   class Store # rubocop:disable Metrics/ClassLength
-    attr_reader :client, :table, :hash_key, :expires_key, :ttl
+    attr_reader :client, :table, :hash_key, :expires_key, :ttl, :unique_process_id
 
-    def initialize(table, hash_key, expires_key: "Expires", ttl: 30, client_options: {})
+    def initialize(table, hash_key, **options)
+      client_options = options[:client_options] || {}
```

Comment:
lets use `fetch` instead of doing the `||` 
```ruby
 options.fetch(:client_options, {})
```
same for the one below (`expires_key` and `ttl`)